guide/popular-topics/miscellaneous-examples.md

```diff
@@ -104,6 +104,9 @@ client.on('message', message => {
 client.login('your-token-goes-here');
 ```
 
+::: tip
+The `escapeRegex` arrow function is a simple function used to remove special characters, that would normally result in an error, from a [regular expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)!
```

Comment:
```suggestion
The `escapeRegex` function is used to convert special characters into literal characters by escaping them, so that they don't terminate the pattern within the [Regular Expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)!
```